app/feedme/zendesk_email_parser.py

```
@@ -0,0 +1,775 @@
+            
+            # Try datetime attribute first
+            datetime_str = time_elem.get('datetime')
+            if datetime_str:
+                return self._parse_datetime(datetime_str)
+            
+            # Try data-timestamp
+            timestamp = time_elem.get('data-timestamp')
+            if timestamp:
+                try:
+                    return datetime.fromtimestamp(int(timestamp))
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)